### ISSUE TITLE
Feature Request: toolbar button and hotkey to switch to currently checked out branch

### ISSUE THREAD

**Issue by dain** [gitx/gitx#412]
I switch branches often on the command line in a project with a lot of branches. It can be a pain to find the currently checked out branch in a long branch list.  Instead, I'd like to have a toolbar button and, more importantly, a hotkey to switch the focus to the currently checked out branch (or raw commit if not on a branch).  This would be the equivalent of quitting and restarting GitX for that project.

**Comment by hannesa2** [comment]
Right, _gitx_ doesn't update branch when you switch branch in command line.
To update _gitx_ I **only click** here in  this area and it will change to right branch immediate 
![image](https://github.com/gitx/gitx/assets/3314607/e2205704-d7bc-4871-9eac-c315793cfe71)


Sorry, this is just a workaround

**Comment by dain** [comment]
That is what I'm doing today.  The problem is the project I'm working on has a couple hundred branches so scanning that list is not fast for me. Having a how key would make my life a lot easier.  Also if the checks out branch was inverse color instead of the circle check would also help. Im a bit colorblind so I have problems finding stuff like that in a long list.
